tests/README.md

```diff
@@ -0,0 +1,515 @@
+# Getting Started with Consensus Spec Tests
+
+## Getting Started
+
+### Creating the environment
```

Comment:
Although is nice, it overlaps somewhat with https://github.com/ethereum/consensus-specs/blob/dev/tests/core/pyspec/README.md

Maybe we can combine them, or update the existing doc and reference it here? What you added here does look good.

Reply:
It does, but that is because a reference and a tutorial on the same subject are going to have some duplication. In a tutorial I need to give people everything to get from a bare OS to being able to do whatever I'm teaching, such as consensus layer tests.

I could update https://github.com/ethereum/consensus-specs/blob/dev/tests/core/pyspec/README.md, but I'd rather also keep all of this material here. Is that OK?

Reply:
I included a link to https://github.com/ethereum/consensus-specs/blob/dev/tests/core/pyspec/README.md.